src/tools/blast/blastcommandline.jl

```diff
@@ -0,0 +1,24 @@
+# wrapper for BLAST+ command line functions
+
+using LightXML
+
+type BlastResult
+    queryname::String
+    subjectname::String
+    query::NucleotideSequence
+    subject::NucleotideSequence
+end
+    
+
+
+"""
+Runs blastn on `query` against `subject`. May include optional `flags` such as
+    `[-pident, 95]`. Do not use `-outfmt`.
+"""
+function blastn(query::String, subject::String, flags...)
+    result = readall(`blastn -query $query -subject $subject -outfmt 5 $flags`)
+    parse_string(result)
+end
+
+#testing
+println(blastn("/Users/KBLaptop/computation/tmp/testseq.fasta", "/Users/KBLaptop/computation/tmp/testseq2.fasta"))
```

Comment:
Agree - I'm going to need a lot of help devising proper tests I think. This is just part of my own development process. Do you know of a good resource that provides some basic principles for developing tests?